specifyweb/frontend/js_src/lib/components/TreeView/index.tsx

```diff
@@ -179,12 +190,13 @@ function TreeView<SCHEMA extends AnyTree>({
         />
         <Button.Icon
           aria-pressed={isSplit}
+          disabled={screenWidth <= 640}
           icon="template"
           title={treeText.splitView()}
           onClick={() => setIsSplit(!isSplit)}
         />
         <Button.Icon
-          disabled={!isSplit}
+          disabled={!isSplit || screenWidth <= 640}
```

Comment:
rename `isSplit` to `rawIsSplit` and create variable `const isSplit = rawIsSplit && canSplit;` and use that variable in these places